docs/layouts/_default/baseof.html

```diff
@@ -38,7 +38,7 @@
   </head>
   <body class="dark:bg-dark font-sans antialiased flex flex-col min-h-full{{ if $hasBottomNav }} pb-16 lg:pb-0{{ end }}" onload="showBody()" style="visibility: hidden;">
     <div class="flex-1 z-1">
-      {{ partial "banner.html" . }}
+      {#{{ partial "banner.html" . }}#}
```

Comment:
Go template commenting needs to look like this:

```html
{{/* {{ partial "banner.html" . }} */}}
```

Reply:
Ah, you are right. I was relying on my vim plugin but it seems to have picked up that syntax as some other templating language.

Reply:
Updated in 399006649